src/eXpansion/Framework/Core/Model/ChatCommand/AbstractChatCommand.php

```diff
@@ -96,7 +96,7 @@ public function run($login, ChatOutput $output, $parameter)
             $parameter = str_getcsv($parameter, " ", '"');
             $parameter = array_merge([0 => 1], $parameter);
 
-            $input = new ArgvInput($parameter);
+            $input = new ArgvInput($parameter, $this->inputDefinition);
```

Comment:
Why ? 

It's done in 2 times for a reason without this --help won't work.

Reply:
ok i see, but then the iterator has a problem. it can't pick other than 1 argument...
` /admin kick toto "reason here"`

Will pass only `/admin kick toto` to the chat command.

Reply:
Ok, I created separate ticket can you revert this it also causes the Unit test to fail